src/byrow.jl

```diff
@@ -28,18 +28,22 @@ byrow_replace(x) = x
 
 function byrow_find_newcols(e::Expr, newcol_decl)
     if e.head == :macrocall && e.args[1] == Symbol("@newcol")
-        ea = e.args[2]
+        if VERSION < v"0.7-"
```

Comment:
Probably https://github.com/JuliaLang/julia/commit/3c3ced4153cdc2c75319b41155c38ad1f2545fc2? That's `0.7.0-DEV.481`.